util/Makefrag

```diff
@@ -83,6 +83,12 @@ define QUESTASIM
 				-t 1ps -voptargs=+acc $1 +permissive-off ++$$1 \
 				-do "log -r /*; run -a"' >> $@
 	@chmod +x $@
+	@echo "#!/bin/bash" > $@.gui
+	@echo '${VSIM} +permissive -work ${VSIM_BUILDDIR} \
+				-ldflags "-Wl,-rpath,${FESVR}/lib -L${FESVR}/lib -lfesvr -lutil" \
+				-t 1ps -voptargs=+acc $1 +permissive-off ++$$1 \
+				-do "log -r /*; run -a"' >> $@.gui
+	@chmod +x $@.gui
```

Comment:
Cool. Can you add this to the README please? Nice feature that people should be aware of :)

Reply:
Good point, done in deb3f6a